src/features/bookings/components/BookingDateTime.tsx

```diff
@@ -13,8 +17,33 @@ export const BookingDateTime = ({
   accessible = false,
   inListItem = false,
 }: Props) => {
+  const { t } = useTranslation();
+
+  const date = booking
+    ? DateTime.fromJSDate(booking?.startsAt, {
+        zone: IANAZone.create('Europe/Rome'),
+      }).toFormat('dd MMMM')
+    : '';
+  const startsAtTime = booking
+    ? DateTime.fromJSDate(booking?.startsAt, {
+        zone: IANAZone.create('Europe/Rome'),
+      }).toFormat('HH:mm')
+    : '';
+  const endAtTime = booking
+    ? DateTime.fromJSDate(booking?.endsAt, {
+        zone: IANAZone.create('Europe/Rome'),
+      }).toFormat('HH:mm')
+    : '';
```

Comment:
don't we have any helper that can do this without all these replicated lines?
@emacoricciati / @FabrizioCostaMedich